src/common/logger/logger_settings.h

```diff
@@ -6,7 +6,11 @@ struct LogSettings
     // The following strings are not localizable
     inline const static std::wstring defaultLogLevel = L"warn";
     inline const static std::wstring logLevelOption = L"logLevel";
-
+    inline const static std::string runnerLoggerName = "runner";
+    inline const static std::wstring runnerLogPath = L"RunnerLogs\\runner-log.txt";
+    inline const static std::string launcherLoggerName = "launcher";
+    inline const static std::wstring launcherLogPath = L"Logs\\launcher-log.txt";
```

Comment:
`AppData\Local\Microsoft\PowerToys\PowerToys Run\Logs` contains the logs from the PR Run core app, we should make it a clear distinction between that log and the log from the module interface dll, so instead of using `Logs` let's use `LogsModuleInterface`

Reply:
done